templates/nginx/galaxy-test.j2

```diff
@@ -79,7 +79,7 @@ server {
 		proxy_set_header         Upgrade $http_upgrade;
 		proxy_set_header         Connection "upgrade";
 		client_max_body_size     0;
-		proxy_pass http://upload.galaxyproject.eu:1080/files;
+		proxy_pass http://upload.galaxyproject.eu:1080/api/upload/resumable_upload;
```

Comment:
Same story as in [this comment](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/782/files#r1218196152).